services/venice-controller/src/main/java/com/linkedin/venice/pushmonitor/AbstractPushMonitor.java

```diff
@@ -755,6 +761,7 @@ public void disableReplica(String instance, int partitionId) {
             kafkaTopic,
             Collections.singletonList(HelixUtils.getPartitionName(kafkaTopic, partitionId)));
         disabledReplicaMap.computeIfAbsent(instance, k -> new HashSet<>()).add(partitionId);
+        disabledPartitionStats.recordDisabledPartition();
```

Comment:
Currently do we reenable any partition? If so, we need to decrement the count?

Reply:
Talked with Sourav offline; reenable will happen after the node restarts; however, the Count metric will also be reset to 0 after node restarts, so there is no need to do decrement with the current code base.

Reply:
since we renable only on restart, it will clear the metric then.